supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -124,7 +124,7 @@
 [discrete]
 [[host-name]]
 ==== host name (noun)
-*Description*: "Host name" is preferred spelling; do not use "hostname." Only capitalize the initial "H" at the beginning of a sentence. See _The IBM Style Guide_ for more information.
+*Description*: "Host name" is preferred spelling; do not use "hostname." Only capitalize the initial "H" at the beginning of a sentence. Note that this is the opposite default guidance from _The IBM Style Guide_, which recommends using "hostname" for new content.
```

Comment:
This is related to #45, which is requesting to point out which items override IBM guidance (which we may or may not want to do).

I'd recommend just removing that last sentence about seeing the ISG.

Reply:
OK, updated.